mex/backend/merged/helpers.py

```diff
@@ -126,26 +161,43 @@ def create_merged_item(
     if rule_set:
         _apply_additive_rule(merged_dict, fields, rule_set.additive)
         _apply_subtractive_rule(merged_dict, fields, rule_set.subtractive)
-    merged_item = reraising(
-        ValidationError,
-        InconsistentGraphError,
-        cls.model_validate,
-        merged_dict,
-    )
-    return cast(AnyMergedModel, merged_item)  # mypy, get a grip!
+    try:
+        return cls.model_validate(merged_dict)
+    except ValidationError as error:
+        raise InconsistentGraphError from error
```

Comment:
This is not necessarily true as the data leading to the broken merged item could have been provided via API. Make it a `MExError` instead?